src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -89,6 +91,8 @@ private SwitchBootstraps() {}
      *       from the target's class; or</li>
      *   <li>the element is of type {@code String} or {@code Integer} and
      *       equals to the target.</li>
+     *   <li>the element is of type {@code EnumDesc}, that describes a constant that is
```

Comment:
I think that at ~line 76, up in this same javadoc you need to add a reference to EnumDesc in the general description. Where it says:

```
The static arguments are an array of case labels which must be non-null and of type
{@code String} or {@code Integer} or {@code Class}.
```

Reply:
Should be resolved by:
https://github.com/openjdk/jdk/pull/13074/commits/bb59254e409821c9b14287c73a35ebe2dd2cbda9

Thanks for the comment!